src/en/credentials.md

```diff
@@ -46,8 +60,8 @@ have more than one for a cloud:
 juju set-default-credential <cloud> <credential>
 ```
 
-If you are having difficulty finding the correct credential information for 
-your cloud, please see the [help pages][help].
+The default credentials do not need to be specified on the commandline for 
+bootstrap (creating a controller).  
```

Comment:
I find this sentence confusing and ill-worded at the end. What are you trying to say here? A default does not need to be specified for **any** command as long as you specify a credential to use (--credential).